### PR TITLE
Update copy for request confirmation email

### PR DESCRIPTION
**Pull request by jamestran201**
Trello card: https://trello.com/c/Bpy2CoBJ/197-when-a-provider-gets-their-request-confirmation-email-they-see-copy-encouraging-them-to-share-with-their-colleagues-social-share

This PR updates the copy of the request created email and add some social media links to it:
Screen_Shot_2020-04-10_at_10 03 28_PM

Also, a mailer preview for the `request_created_email` is added to make it easier to view email changes.

To-do:
- [x] Confirm that the social media icons are ok with Design

### CODE REVIEW

File: app/views/provider_mailer/request_created_email.erb [hunk 1]
```diff
@@ -8,9 +8,34 @@
...
 <p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">Volunteers will reach out to you via this link to offer and coordinate help.</p>
-<p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">Your request will expire in 14 days.</p>
 <p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">You can edit or delete your request at any time by clicking the link below.</p>
 <p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;"><a href="<%= @edit_url %>"><%= @edit_url %></a></p>
-<p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">We truly appreciate your service. Please reach out to us if you have any other suggestions for other ways we can help you.</p>
+<p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">Your request will expire in 14 days.</p>
+<p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">Please spread the word to your colleagues and friends so we can help support everyone you work with!</p>
+<table align="center" style="width: 50%">
```
Comment (tuntisz) [inline app/views/provider_mailer/request_created_email.erb:15]: Nice work here :beers: email layout is a pain

File: spec/mailers/previews/provider_mailer_preview.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ProviderMailerPreview < ActionMailer::Preview
+  def request_created_email
```
Comment (tuntisz) [inline spec/mailers/previews/provider_mailer_preview.rb:4]: Awesome!

**Review by tuntisz — APPROVED** [review]
Looks great!

**Comment by jamestran201** [comment]
@tuntisz Thanks!

**Pull request merged**